types/find/index.d.ts

```diff
@@ -0,0 +1,116 @@
+// Type definitions for find 0.2
+// Project: https://github.com/yuanchuan/find#readme
+// Definitions by: Andrey Lalev <https://github.com/andypyrope>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface AsyncFindStream {
+    /**
+     * Handling errors in asynchronous interfaces.
+     * @param callback The callback that is called upon an error
+     */
+    error(callback: (() => void)  | ((err: Error) => void)): void;
+}
+
+export interface FindEachStream {
+    /**
+     * Handling errors in asynchronous interfaces.
+     * @param callback The callback that is called upon an error
+     */
+    error(callback: (() => void) | ((err: Error) => void)): FindEachStream;
+
+    /**
+     * Detect end in find.eachfile and find.eachdir.
+     * @param callback The callback called at the end of find.eachfile and find.eachdir
+     */
+    end(callback: () => void): FindEachStream;
+}
+
+/**
+ * Find all files in a given directory asynchronously.
+ * @param root The root directory
+ * @param callback A callback that accepts an array of the found files
+ */
+export function file(root: string, callback: (files: string[]) => void): AsyncFindStream;
+
+/**
+ * Find all files that match a glob pattern in a given directory asynchronously.
+ * @param pattern The pattern to filter the files with
+ * @param root The root directory
+ * @param callback A callback that accepts an array of the found files
+ */
+export function file(pattern: string | RegExp, root: string, callback: (files: string[]) => void): AsyncFindStream;
+
+/**
+ * Find all files in a given directory asynchronously.
+ * @param root The root directory
+ * @param callback A callback that accepts each file separately
+ */
+export function eachFile(root: string, callback: (file: string) => void): FindEachStream;
+
+/**
+ * Find all files that match a glob pattern in a given directory asynchronously.
+ * @param pattern The pattern to filter the files with
+ * @param root The root directory
+ * @param callback A callback that accepts an array of the found files
+ */
+export function eachFile(pattern: string | RegExp, root: string, callback: (file: string) => void): FindEachStream;
```

Comment:
is this supposed to be `eachfile` or `eachFile`?

Reply:
Feel free to send a followup fix PR.